models/gotbook.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Gotbook = sequelize.define('Gotbook', {
```

Comment:
Gotbooks, bad model name. It should be ReaderBookRequest model based on what I read below